### PR TITLE
#85 -  fix bug for show content a other browsers

### PR DESCRIPTION
**Pull request by MrHDOLEK**
Should close https://github.com/blumilksoftware/meetup/issues/85

### CODE REVIEW

**Review by JakubZakowicz — APPROVED** [review]

File: src/Http/Controllers/StaticController.php [hunk 1]
```diff
@@ -22,9 +23,8 @@ public function index(Request $request): BinaryFileResponse
         if (!File::exists($path) || $this->filesFilter(File::basename($path))) {
             abort(Response::HTTP_NOT_FOUND);
         }
-
         $headers = [
-            "Content-Type" => File::type($path),
+            "Content-Type" => File::extension($path) === self::FILE_CSS_EXTENSION ? null : File::type($path),
```
Comment (krzysztofrewak) [inline src/Http/Controllers/StaticController.php:27]:
```suggestion
            "Content-Type" => File::extension($path) === static::FILE_CSS_EXTENSION ? null : File::type($path),
```

Comment (krzysztofrewak) [reply src/Http/Controllers/StaticController.php:27]: Also: I don't like narrowing to for CSS files only. Maybe renamed const should be an array of strings/types?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/StaticController.php [hunk 2]
```diff
@@ -11,6 +11,9 @@
 
 class StaticController extends Controller
 {
+    private const FILE_EXTENSION = [
```
Comment (krzysztofrewak) [inline src/Http/Controllers/StaticController.php:14]: Oh, come on, this constant name doesn't tell anything about its content. Please make it more verbose.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/StaticController.php [hunk 3]
```diff
@@ -11,6 +11,9 @@
 
 class StaticController extends Controller
 {
+    private const IGNORE_HEADER_FILES_EXTENSION = [
```
Comment (krzysztofrewak) [inline src/Http/Controllers/StaticController.php:14]:
```suggestion
    private const IGNORED_HEADER_FILES_EXTENSIONS = [
```

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
IGNORED_FILES would be nice too.

**Review by krzysztofrewak — APPROVED** [review]

**Review by JakubZakowicz — APPROVED** [review]

**Pull request merged**